installinator/src/fetch.rs

```diff
@@ -227,6 +227,7 @@ impl FetchArtifactBackend {
             InstallinatorProgressMetadata::Download { peer: peer.address() };
 
         loop {
+            // This is the read timeout.
```

Comment:
Small nit - this seems strongly related to the block comment in `artifact.rs`, which is easy to see on this PR but is a lot less obvious if just opening `fetch.rs`. Maybe either refer to that comment or expand this one a bit to give more context?

Reply:
added a reference to that block, thanks!